src/chain.h

```diff
@@ -200,7 +200,7 @@ class CBlockIndex
 
     //! (memory only) Number of transactions in the chain up to and including this block.
     //! This value will be non-zero only if and only if transactions for this block and all its parents are available.
-    //! Change to 64-bit type when necessary; won't happen before 2030
+    //! Change to 64-bit type before 2024-2030, depending on SegWit adoption.
```

Comment:
I'd rather just put the worst-worst case here, which is 60 bytes per tx. Since it seems unlikely in the first place that everyone would be using 1-in-1-out txs with just OP_TRUE as witness.

```
    4 bytes version
    1 byte input count
        36 bytes outpoint
        1 byte scriptSigLen (0x00)
        0 bytes scriptSig
        4 bytes sequence
    1 byte output count
        8 bytes value
        1 byte scriptPubKeyLen
        1 byte scriptPubKey (OP_TRUE)
    4 bytes locktime

Reply:
Rebased, and updated PR description and source comment.